ee/app/livechat-enterprise/server/hooks/findDepartmentsWithBusinessUnits.ts

```diff
@@ -0,0 +1,15 @@
+import { LivechatDepartment } from '../../../../../app/models/server';
+import { callbacks } from '../../../../../lib/callbacks';
+
+callbacks.add(
+	'livechat.findDepartmentsWithBusinessUnits',
+	async ({ businessUnitIds } = { businessUnitIds: [] }) => {
+		console.log('enterprise code called', businessUnitIds);
+
+		return LivechatDepartment.findByUnitIds(businessUnitIds, {
```

Comment:
What if the BU doesn't exist? We need to verify/validate before reading departments.